src/targeting/Find_targets.ml

```diff
@@ -309,8 +324,10 @@ let group_scanning_roots_by_project (conf : conf)
     (scanning_roots : Fpath.t list) : project_roots list =
   let force_root =
     match conf.project_root with
-    | None -> None
-    | Some proj_root -> Some (Project.Gitignore_project, proj_root)
+    | Some (Git_remote _)
+    | None ->
+        None
+    | Some (Filesystem proj_root) -> Some (Project.Git_project, proj_root)
```

Comment:
yea I tried that initially, but realized this is only for `force_root` which forcibly relativizes scan roots to project roots.

I.e. if the project_root is `/repo/src/` and the scanning root is `/src/foo` it would make the scanning root `/foo`. So it doesn't make sense to do this with the git remote unless we wanted to make it so git remotes could be further specified (say github.com/semgrep/semgrep.git:/src/foo).

Reply:
I'll add some comments explaining this because I also thought similarly when I first saw this

Reply:
Oops, I also changed the project type